sdcm/provision/common/utils.py

```diff
@@ -128,6 +128,10 @@ def install_syslogng_service():
     return dedent("""\
         if yum --help 2>/dev/null 1>&2 ; then
             if ! rpm -q syslog-ng ; then
+                for n in {1..9}; do
+                    yum install -y --downloadonly syslog-ng && break
```

Comment:
Won't it exit script with the non-zero code if we fail to download it?

Reply:
Yes and no, It will be spinning 9 times then stop on `yum install -y syslog-ng` which will give you $? == 1 , which is exactly what we want.